utils/windowfunction/wf_stats.h

```diff
@@ -40,8 +40,8 @@ class WF_stats : public WindowFunctionType
   static boost::shared_ptr<WindowFunctionType> makeFunction(int, const string&, int, WindowFunctionColumn*);
 
  protected:
-  long double fSum1;
-  long double fSum2;
+  long double fMean;
```

Comment:
According with the styling guide new attributes must be prefixed with _ thus fM2sum_ is the name.